client/src/components/DownloadStarted.js

```diff
@@ -22,6 +23,7 @@ export const DownloadStarted = ({
   const { items, info, type, isProject } =
     getDownloadOptionDetails(computedFile)
   const additionalRestrictions = resource.additional_restrictions
+  const isMergedObjects = computedFile.s3_key.includes('merged')
```

Comment:
Please check `computedFile.is_merged` here, we shouldn't inspect the `s3_key`.

Reply:
Could you please tell me about the `is_merged` flag (meant as `includes_merged`)?

Reply:
Done at b62074563a5c3718b9c209c9286e536d65a66021 (thank you for the quick fix #654 🚀 )!